extensions/scarb-snforge-test-collector/tests/test.rs

```diff
@@ -48,10 +48,7 @@ fn forge_test_locations() {
     assert_eq!(&json[1]["test_cases"][0]["name"], "tests::tests::test");
     assert_eq!(&json[1]["tests_location"], "Tests");
 
-    assert_eq!(
-        &json[0]["test_cases"][0]["available_gas"],
-        &Value::Number(Number::from(u32::MAX))
-    );
+    assert_eq!(&json[0]["test_cases"][0]["available_gas"], &Value::Null);
```

Comment:
Would be nice to have a test when `available_gas` is set to the highest possible value

Reply:
It's out of scope for this PR IMO